applications/yolo_model_deployment/CMakeLists.txt

```diff
@@ -53,8 +49,6 @@ add_custom_command(OUTPUT "${HOLOHUB_DATA_DIR}/yolo_model_deployment/meeting.gxf
     COMMAND ffmpeg -i "${HOLOHUB_DATA_DIR}/yolo_model_deployment/meeting.mp4" -pix_fmt rgb24 -f rawvideo pipe:1 |
             python3 "${CMAKE_SOURCE_DIR}/utilities/convert_video_to_gxf_entities.py"
             --directory "${HOLOHUB_DATA_DIR}/yolo_model_deployment" --basename meeting --width 1280 --height 720 --framerate 25
-    BYPRODUCTS "${HOLOHUB_DATA_DIR}/yolo_model_deployment/meeting.gxf_index"
-               "${HOLOHUB_DATA_DIR}/yolo_model_deployment/meeting.gxf_entities"
```

Comment:
Hi @tbirdso, I've used the same commands for preparing this data source for vila live, guess the same issue will appear here as well: https://github.com/nvidia-holoscan/holohub/blob/97197b08c60fecdbf8ab3f99eb2142e7e02fa437/applications/vila_live/CMakeLists.txt#L40-L41

Reply:
Good catch @wyli , opened https://github.com/nvidia-holoscan/holohub/pull/746 with this fix. There is an additional runtime error in VILA Live requiring further attention.